games-strategy/1oom/1oom-1.0.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="Open source clone of Master of Orion"
+DESCRIPTION="1oom is a game engine recreation of the original \"Master \
+of Orion\" (1993), a 4X strategy space game originally made by Simtex \
+and released by Microprose.
+
+Please note you will need the original game LBX data files from version 1.3. \
+This means that you will need a purchased copy of Master of Orion \
+in order to use 1oom. \
+
+To play: copy the LBX files into /boot/home/config/non-packaged/data/1oom
+
+Configuration may be changed by editing the 1oom_config_game_classic_sdl1.txt \
+file, located in /boot/home/config/settings/1oom"
+HOMEPAGE="https://kilgoretroutmaskreplicant.gitlab.io/plain-html/"
+COPYRIGHT="2018-2021 1oom developers"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://gitlab.com/Tapani_/1oom/-/archive/master/1oom-master.tar.gz?sha=81ce4060661fa724fa97e12d97e39c8c39b9deb6"
+CHECKSUM_SHA256="5538253ec444041955c11c88b73ec62764359238e145afa4508e6ba58fb410d8"
+SOURCE_FILENAME="1oom-master.tar.gz"
+SOURCE_DIR="1oom-master"
+PATCHES="1oom-$portVersion.patchset"
+ADDITIONAL_FILES="1oom.rdef"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	1oom$secondaryArchSuffix = $portVersion
+	app:1oom$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libSDL$secondaryArchSuffix
+	lib:libSDL_mixer$secondaryArchSuffix
+	lib:libsamplerate$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libglu$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
```

Comment:
please make this list alphabetical keeping haiku${secondaryArchSuffix}_devel at the top